db/migrate/20200126203714_remove_vim_types_from_ems_events.rb

```diff
@@ -0,0 +1,86 @@
+class RemoveVimTypesFromEmsEvents < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+  include MigrationHelper
+
+  BATCH_SIZE = 5_000
+
+  module VimType
+    def vimType
+      @vimType.nil? ? nil : @vimType.to_s
```

Comment:
Minor, but `@vimType&.to_s` will do the same (even though nil responds to .to_s, `&.` short circuits it anyway)

Reply:
I copied exactly what was in `VMwareWebService/VimTypes` but I can change this if you want

Reply:
Nah this is fine.